crates/nu-command/src/filesystem/glob.rs

```diff
@@ -99,6 +99,8 @@ impl Command for Glob {
         let glob_pattern: Spanned<String> = call.req(engine_state, stack, 0)?;
         let depth = call.get_flag(engine_state, stack, "depth")?;
 
+        if glob_pattern.item.is_empty() { return Ok(PipelineData::new(Span::new(0, 0))); }
```

Comment:
The `Span::new(0, 0)` doesn't convey helpful information if you want to backtrack helpful information for error handling. The `span` in scope from `call.head` will point to the command at hand `glob` (not at the argument, but that is up for debate what is more useful here.)